packages/functional-tests/pages/settings/recoveryPhone.ts

```diff
@@ -39,7 +39,7 @@ export class RecoveryPhoneSetupPage extends SettingsLayout {
     await this.sendCodeButton.click();
   }
 
-  async expectAddErrorBanner(message: string) {
+  async expectAddErrorBanner(message: string | RegExp) {
```

Comment:
Best practice is to keep expects/assertions in tests and not POM, so this `expectAddErrorBanner` could be removed from page model.